test/server/cards/03_TWI/units/RoyalGuardAttache.spec.ts

```diff
@@ -0,0 +1,24 @@
+describe('Royal Guard Attaché', function () {
+    integration(function (contextRef) {
+        describe('Royal Guard Attaché\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['royal-guard-attache'],
+                    },
+                    player2: {
+                        groundArena: ['death-trooper'],
```

Comment:
Our practice is to only include things in the test setup that are relevant to the test just for readability, in this case we could remove Death Trooper and the entire `player2` object since there's nothing else in there